src/main/java/com/marketdata/MarketSimulatorBackend/api/controller/StockController.java

```diff
@@ -13,19 +15,18 @@
 @RestController
 @RequestMapping("/api")
```

Comment:
It's a good practice to add the api version as well, so this becomes "api/v1" so later you can release multiple versions if needed. 

I suggest that you add swagger annotation as well, it will make you able to generate swagger documentation of the API automatically which will be very helpful in testing and documentation.